eos/b-decays/bs-to-kstar-l-nu.cc

```diff
@@ -605,7 +611,7 @@ namespace eos
     {
         AngularCoefficients a_c = _imp->bstokstarlnu._imp->integrated_angular_coefficients(0.02, 19.71);
 
-        return 4.0 / 9.0 * (2.0 * a_c.j1s + 3.0 * a_c.j3) / _imp->btopilnu.integrated_decay_width(0.02, 12.0);
+        return 4.0 / 9.0 * (2.0 * a_c.j1s + 3.0 * a_c.j3) * _imp->tau() / _imp->hbar() / _imp->btopilnu.integrated_branching_ratio(0.02, 12.0);
```

Comment:
Please check whether you need to normalize to B -> Pi mu nu only in "ratio_perp" or also in the other two ratios "ratio_para" and "ratio_long". Also seems strange convention to integrate B_s -> K^* mu nu up to 19.71 GeV², but the
 B -> Pi mu nu only up to 12 GeV^2 ?!

Reply:
That was on purpose for the project, see http://arxiv.org/abs/arXiv:1503.09063